app/grandchallenge/challenges/models.py

```diff
@@ -1402,3 +1403,81 @@ class ChallengeRequestGroupObjectPermission(GroupObjectPermissionBase):
     content_object = models.ForeignKey(
         ChallengeRequest, on_delete=models.CASCADE
     )
+
+
+class TaskResponsibleChoices(models.TextChoices):
+    SUPPORT = "SUP", "Support"
+    CHALLENGE_ORGANIZERS = "ORG", "Challenge Organizers"
+
+
+class OnboardingTask(FieldChangeMixin, UUIDModel):
+    ResponsibleChoices = TaskResponsibleChoices
+
+    class Meta:
+        permissions = [
+            ("complete_onboaringtask", "Can mark this task as completed")
+        ]
+
+    created = models.DateTimeField(editable=False)
+    challenge = models.ForeignKey(
+        to=Challenge,
+        on_delete=models.CASCADE,
+        related_name="onboarding_tasks",
+    )
+    title = models.CharField(
+        max_length=255,
+        help_text="Title of this task",
+    )
+
+    description = models.TextField(
+        blank=True, help_text="Description of this task."
+    )
+    complete = models.BooleanField(
+        default=False,
+        help_text="If true, this task is complete.",
+    )
+    completed_at = models.DateTimeField(
+        null=True,
+        blank=True,
+        help_text="Time this task was last marked completed.",
+    )
+    responsible = models.CharField(
+        blank=False,
+        max_length=3,
+        choices=ResponsibleChoices.choices,
+        default=ResponsibleChoices.CHALLENGE_ORGANIZERS,
+        help_text="Who is responsible for completion of this task.",
+    )
```

Comment:
Maybe call this `assigned_to` or `responsible_group`, rather than `responsible`?

Reply:
I agree it's better to change it. I would like to avoid user or group naming since it could suggest an actual implemented user or group. How about `responsible_party`?

Reply:
I see why you want to avoid `responsible_group`. What is wrong with `assigned_to` - that feels neutral and clear to me. But `responsible_party` is also fine with me.